libshaderc/README.md

```diff
@@ -0,0 +1,43 @@
+# libshaderc
+
+A library for compiling shader strings into SPIR-V.
+
+## Build Artifacts
+There are two main build artifacts that are created from a CMake build of
+libshaderc.
+The first is libshaderc which is a static library containing just the
+functionality needed. This has static dependencies on other libraries compiled
+during the build process. `glslang OSDependent OGLCompiler shaderc_util SPIRV`.
+
+The other is libshaderc_combined which is a static library containing libshaderc
+and all of it's dependencies.
+
+
+## Integrating libshaderc
+
+There are several ways of integrating libshaderc into external projects.
+
+1.  If the external project uses CMake, then `shaderc/CMakeLists.txt` can be
+included into the file and `target_link_libraries(shaderc)` can be specified.
```

Comment:
then `shaderc/CMakeLists.txt` can be included into the external project's CMake configuration ... specified to link against libshaderc.